privateid_test.go

```diff
@@ -0,0 +1,16 @@
+package goinsta
+
+import (
+	"testing"
+)
+
+func TestMediaIDFromPrivateID(t *testing.T) {
+	mediaID, err := MediaIDFromPrivateID("CFPU5jqA7F7gtdLtpBevHl5A6rVwt3fi4zVzs00")
```

Comment:
Also, add more tests here to make sure about the function. For example, test invalid codes.